submissions/chabuChabu/php/weather.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+$temp = 33;
+
+if($temp <= 20){
+    echo"It is freezing cold today";
+}elseif($temp > 20 && $temp < 30){
+    echo"The weather is just perfect";
+}elseif($temp > 30 and $temp < 40){
```

Comment:
Well done. You didn't fulfill the criteria for this part. Go through the criteria again and make the necessary corrections.

Reply:
Correction now done.Thank you